mods/ts/rules/palettes.yaml

```diff
@@ -22,7 +22,6 @@
 		Name: chrome
 		Filename: cameo.pal
 		AllowModifiers: false
-		ShadowIndex: 242
```

Comment:
Sadly this breaks the clock:

![image](https://cloud.githubusercontent.com/assets/756669/9040116/355bf4a4-3a00-11e5-99a1-ff5eaab50762.png)